action/protocol/poll/protocol.go

```diff
@@ -58,18 +57,13 @@ type lifeLongDelegatesProtocol struct {
 }
 
 // NewLifeLongDelegatesProtocol creates a poll protocol with life long delegates
-func NewLifeLongDelegatesProtocol(delegates []genesis.Delegate) Protocol {
+func NewLifeLongDelegatesProtocol(addrs []address.Address) Protocol {
```

Comment:
with votes and reward addresses, ppl could easily use lifelongdelegatesprotocol to start a consortium chain with fixed portion for each participate, or some other purpose. no hurt to keep this